hypervisor/dm/vpci/vpci.c

```diff
@@ -495,8 +495,10 @@ static int32_t write_pt_dev_cfg(struct pci_vdev *vdev, uint32_t offset,
 		write_sriov_cap_reg(vdev, offset, bytes, val);
 	} else {
 		if (!is_quirk_ptdev(vdev)) {
-			/* passthru to physical device */
-			pci_pdev_write_cfg(vdev->pdev->bdf, offset, bytes, val);
+			if (offset != vdev->pdev->sriov.pre_pos) {
+				/* passthru to physical device */
+				pci_pdev_write_cfg(vdev->pdev->bdf, offset, bytes, val);
+			}
```

Comment:
Lifei said this is incorrect. We need to do this before if (!is_quirk_ptdev()..
I am fine with either. Please check with Fei, and ask him to review.

Reply:
Li Fei has see the PR patch, he reviewed and ask me to PR, I will ask him for this issue

Reply:
He said should before  if (!is_quirk_ptdev(), will fix that

Reply:
fixed